reversion/models.py

```diff
@@ -62,6 +64,47 @@ class Revision(models.Model):
         help_text="A text comment on this revision.",
     )
 
+    def get_comment(self):
+        """
+        A Copy of django's admin.LogEntry `get_change_message` as we're keeping the same JSON change message of
+        django's admin.
+        Issue: 725
+        """
+        if self.comment and self.comment[0] == '[':
+            try:
+                change_message = json.loads(self.comment)
+            except ValueError:
+                return self.comment
+            messages = []
+            for sub_message in change_message:
+                if 'added' in sub_message:
+                    if sub_message['added']:
+                        sub_message['added']['name'] = ugettext(sub_message['added']['name'])
+                        messages.append(ugettext('Added {name} "{object}".').format(**sub_message['added']))
+                    else:
+                        messages.append(ugettext('Added.'))
+
+                elif 'changed' in sub_message:
+                    sub_message['changed']['fields'] = get_text_list(
+                        sub_message['changed']['fields'], ugettext('and')
+                    )
+                    if 'name' in sub_message['changed']:
+                        sub_message['changed']['name'] = ugettext(sub_message['changed']['name'])
+                        messages.append(ugettext('Changed {fields} for {name} "{object}".').format(
+                            **sub_message['changed']
+                        ))
+                    else:
+                        messages.append(ugettext('Changed {fields}.').format(**sub_message['changed']))
+
+                elif 'deleted' in sub_message:
+                    sub_message['deleted']['name'] = ugettext(sub_message['deleted']['name'])
+                    messages.append(ugettext('Deleted {name} "{object}".').format(**sub_message['deleted']))
+
+            change_message = ' '.join(msg[0].upper() + msg[1:] for msg in messages)
+            return change_message or ugettext('No fields changed.')
+        else:
+            return self.comment
+
```

Comment:
I'm not convinced by copying and pasting the code here.

How about you do this:

``` py
LogEntry(change_message=json.dumps(comment)).get_change_message())
```

This will re-use the existing code from Django quite cleanly.

Reply:
Yes,  absolutely right.
Updating